safe-global/web-core
# fix: migrate widgets to the governance app

## What it solves

Resolves migration to #1383.

## How this PR fixes it

The header and governance section widgets have been migrated, as well as ranked Safe Apps showing the governance app instead of the claiming app.

## How to test it

The governance app has yet to pass QA. After passing, it needs to be listed as a Safe App before the following can be done:

Open the dashboard:
1. Observe that the Safe token widget opens the governance app on click.
2. Observe an unchanged governance section that reacts to dark mode preferences.
3. Observe the governance app ranked first.

## Analytics changes

Clicking the token hidget dispatches a new event, "Open Safe Governance App from widget". (Previously "Open Safe Claiming App from widget".)
